Marlin/src/MarlinCore.cpp

```diff
@@ -1220,6 +1224,15 @@ void setup() {
     SETUP_RUN(ui.reset_status());     // Load welcome message early. (Retained if no errors exist.)
   #endif
 
+  #if PIN_EXISTS(SAFE_POWER)
+    #if ENABLED(HAS_DRIVER_SAFE_POWER_PROTECT)
```

Comment:
I think that when we use HAS_SOMETHING, we don't use ENABLED. The HAS_SOMETHING usually has a integer value 1 to work direct with #if